apps/explorer/src/app/components/blocks/blocks-refetch.tsx

```diff
@@ -11,7 +11,12 @@ export const BlocksRefetch = ({ refetch }: BlocksRefetchProps) => {
   };
 
   return (
-    <Button onClick={refresh} data-testid="refresh" size="xs">
+    <Button
+      onClick={refresh}
+      data-testid="refresh"
+      size="xs"
+      className="dark:text-white"
```

Comment:
You probably dont need (or at least shouldn't need this class)

Reply:
These were added to fix bits that were broken by a previous conversion. If you're really against me adding them, I will remove them from this PR and do it separately.

Reply:
Agreed in person to do this as a follow up